### DISCUSSION TITLE
No saved maps after full cleanup and set up of rooms

### DISCUSSION THREAD

**Discussion by ando01** [rand256/valetudo#524] [Q&A]
I have a Roborock S5Max and ran a full cleanup to get the maps created. I then set up the zones and restricted ares and went into settings to enable the Auto Map Restore feature. Once I click on "Select Map" a message is shown at the bottom of the screen that says "No Saved Maps!" Can someone help me understand why this is happening? 
![Screenshot from 2024-07-22 11-31-53](https://github.com/user-attachments/assets/33aa897b-d746-4425-8891-06da0658ec0d)

**Comment by pidator** [comment]
You'll have to "save the map inside of Valetudo RE" first, before you'll be able to select one. Therefore use the button on main tab (see [here](https://github.com/rand256/valetudo/issues/41#issuecomment-869206714).

**Reply by ando01** [reply]
@pidator I don't seem to have the Load and Store Map buttons. 
![Screenshot from 2024-07-22 14-47-45](https://github.com/user-attachments/assets/3e1abe73-41c8-438d-a2c8-68fff3570179)

**Reply by pidator** [reply]
> You'll first have to enable the feature in settings / web interface / show multimap buttons, then you'll get two new buttons on home tab

It's right above the screenshot ;)

**Reply by ando01** [reply]
@pidator Doh! Now I see it! Thank you for the info :-)
